README.md

```diff
@@ -74,7 +84,7 @@ To compile any of the examples run:
 $ cargo run --example name_of_example
 ```
 
-All available examples are listed under [examples][ex].
+All available examples are listed under [examples][exr].
```

Comment:
Shouldn't that be linking to the examples in master? Maybe someone will think be on the master github and think the link points to master too.

Reply:
Idk, I just left that one as it was.